tun/src/options.rs

```diff
@@ -0,0 +1,38 @@
+use fehler::throws;
+use std::io::Error;
+
+use super::TunInterface;
+
+#[derive(Default)]
+pub struct TunInterfaceOptions {
```

Comment:
How does `TunOptions` sound? Just a bit shorter, I think the interface part could be inferred

Reply:
Yep, `TunOptions` sounds more concise.